docs/source/commandline/commandline-all.rst

```diff
@@ -6,13 +6,17 @@ Universal Commandline Options
 
 This section contains configuration options that are applicable to *all* MAME sub-builds (both SDL and Windows native).
 
-Core commands
+Core Commands
 -------------
 
+.. _mame-commandline-help:
+
 **-help** / **-h** / **-?**
 
 	Displays current MAME version and copyright notice.
 
+.. _mame-commandline-validate:
+
```

Comment:
This can accept a system name or glob pattern.  If you give it a system name or pattern it will just validate matching systems.  If you give it no system name or glob pattern, it will validate all systems and additionally validate all devices.

Reply:
I'm going to start commenting here to keep track of progress and to address questions:

Got this fixed on my side.